hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/Sinks.java

```diff
@@ -939,4 +994,49 @@ private Sinks() {
     ) {
         return Sinks.jdbc(updateQuery, () -> DriverManager.getConnection(connectionUrl), bindFn);
     }
+
+    /**
+     * Returns a sink which publishes values into an
```

Comment:
this javadoc needs almost a complete write. First the lifecycle should be described, and then caveats part. I fount it very confusing to understand what happens when multiple jobs publish to it.